source/labyrinths/castleofthecelestialknights.js

```diff
@@ -266,7 +266,7 @@ module.exports = new LabyrinthTemplate("Castle of the Celestial Knights",
 	},
 	{
 		// Rooms
-		"Event": ["Door 1 or Door 2?", "Twin Pedestals", "Imp Contract Faire", "Gear Collector", "The Score Beggar", "Apple Pie Wishing Well", "Workshop", "Merchant", "Rest Site", "Library", "Treasure"],
+		"Event": ["Door 1 or Door 2?", "Twin Pedestals", "Imp Contract Faire", "Gear Collector", "The Score Beggar", "Apple Pie Wishing Well", "Workshop", "Merchant", "Rest Site", "Library", "Treasure", "Let Sleeping Dogs Lie"],
```

Comment:
```suggestion
		"Event": ["Let Sleeping Dogs Lie", "Door 1 or Door 2?", "Twin Pedestals", "Imp Contract Faire", "Gear Collector", "The Score Beggar", "Apple Pie Wishing Well", "Workshop", "Merchant", "Rest Site", "Library", "Treasure"],
```
We haven't had instance to show this convention yet, but I think it'll be easier in future balance passes if the rooms are ordered:
1. Labyrinth specific rooms
2. Individual rooms that appear in multiple labyrinths
3. Room categories